guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/repository/wizard/CreateRepositoryWizardTest.java

```diff
@@ -569,6 +573,230 @@ public void testManagedRepositoryMultiCompletedClashingGAVForcedTest() {
     }
 
     @Test
+    @SuppressWarnings("unchecked")
+    public void testManagedRepositoryMultiCompletedClashingGAVBeforeRepositoryInitializationTest() {
```

Comment:
Thank you, @manstis. :-)
(Seriously - thanks for the tests. This insane, yet correct, method name is just the cherry on top.)